docs/dashboards.md

```diff
@@ -58,6 +58,8 @@ In create/edit mode, you can add or change the dashboard title, description, das
 
 ![](resources/images/dashboard-create-mode.png)
 
+In some DHIS2 Core instances it is possible to either create a regular (internal) dashboard, or to embed a dashboard from an external source. See [this section](#creating-and-editing-an-external-dashboard) for more information about this.
```

Comment:
I think this info should be emphasized as boxed text somehow, as an aside.

```
Note: if the system has been configured to support embedded dashboards, then you will have the option of creating a regular (internal) dashboard or embedding a dashboard from an external source. See [this section](#creating-and-editing-an-external-dashboard) for more information.
```

Reply:
We discussed this on Slack and concluded we would use the sentence @jenniferarnesen suggested but without the `Note:` prefix, because in the DHIS2 Core docs Notes are typically in emphasised blocks and we do not want to emphasise this too much, since it is a feature which is not often available.